Lib/test/test_asyncio/test_tasks.py

```diff
@@ -3220,6 +3220,38 @@ def coro():
         self.loop.run_until_complete(coro())
         self.assertEqual(result, 11)
 
+    def test_loop_argument_is_deprecated(self):
+        # this test should be removed in Python 4.0 when the loop argument
```

Comment:
If possible, would you like to shorten the 2 line comment to 1 line in these tests? 

Suggestion: Remove test when loop argument is removed in Python 4.0

Thanks for writing tests :smile:

Reply:
Hi @willingc Thanks for your comments. I will change now.